### DISCUSSION TITLE
CLI v32.3.1 is breaking previously working Remix routes

### DISCUSSION THREAD

**Discussion by cill-i-am** [vercel/vercel#10604] [Help]
### Question

We have started getting 404s for two routes on v32.3.1 that were working fine on v32.2.5. The build passes and it looks like the route generates, but the :publicKey/:planId route is throwing a 404 on Vercel, even though it works fine locally.

v32.2.5:
<img width="1207" alt="Screenshot 2023-09-26 at 22 19 51" src="https://github.com/vercel/vercel/assets/8224386/127ef771-0f04-4d1c-a9e8-958224788f64">

and working route: https://test-embedded-health-2bjekie33-go-yonder.vercel.app/health/ilh/20JZFNmkAC7IePmcMXBbHdhxnmWW06R6q8PLAbTPRtEqUcHy/162H3D1?parentDomainName=www.yondertechnology.ltd&theme_primary=%235A2293&startDate=26%2F09%2F2023&adults=1&levelOfCover=2&hospitalExcess=300&dayToDayBenefits=false

v32.3.1:
<img width="1207" alt="Screenshot 2023-09-26 at 22 22 07" src="https://github.com/vercel/vercel/assets/8224386/5b085332-e372-457c-99dd-b1072376bba3">

and now broken route: https://test-embedded-health-j03tv5dpr-go-yonder.vercel.app/health/ilh/20JZFNmkAC7IePmcMXBbHdhxnmWW06R6q8PLAbTPRtEqUcHy/162H3D1?parentDomainName=www.yondertechnology.ltd&theme_primary=%235A2293&startDate=26%2F09%2F2023&adults=1&levelOfCover=2&hospitalExcess=300&dayToDayBenefits=false

**Comment by cill-i-am** [comment]
Follow up: It looks like the cli isn't picking up index routes correctly. Renaming the index route to details and linking to it works. But the above issue remains

**Comment by TooTallNate** [comment]
Hey @Rocinante89, are you still having an issue with this? Would you be able to provide a reproduction so I can take a closer look?

**Comment by cill-i-am** [comment]
Hey Nathan!

No, it was fixed with the latest CLI version.
